src/components/HorizontalCard/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+
+import styles from './HorizontalCard.module.scss'
+
+import classNames from 'classnames'
+
+export interface HorizontalCardProps
+  extends React.HTMLAttributes<HTMLDivElement> {
+  gameImage?: React.JSX.Element
+  imageUrl?: string
+  children?: React.ReactNode
+  title: string
+  orderNumber?: string
+  onCardClick: React.MouseEventHandler<HTMLDivElement>
+  tone?: 'brand' | 'neutral'
+  size?: 'large' | 'small'
+}
+
+const HorizontalCard: React.FC<HorizontalCardProps> = ({
+  gameImage,
+  title,
+  children,
+  orderNumber,
+  onCardClick,
+  tone = 'neutral',
+  size = 'large'
+}) => {
+  const image = gameImage || <img src="/fallback_card.jpg" alt="Game Image" />
+
+  return (
+    <>
+      <div
+        className={classNames(
+          styles.horizontalCard,
+          styles[tone],
+          styles[size]
+        )}
+        onClick={onCardClick}
```

Comment:
This way we're more un-opinionated and extensible with the component styling

```suggestion
        onClick={onCardClick}
        {...props}
```